src/util.js

```diff
@@ -430,6 +430,14 @@ var _ = $.extend(Mavo, {
 		return element.getAttributeNames().filter(name => regex.test(name));
 	},
 
+	// Fixes the case of attributes that are not all lowercase
+	// Especially useful for SVG attributes
+	// https://html.spec.whatwg.org/multipage/parsing.html#adjust-svg-attributes
+	getProperCasing: (element, attribute, root = "svg") => {
```

Comment:
1. You don't seem to be using the 3rd argument anywhere. Perhaps a better design might be to pass in an actual element, and get its root via `.closest()` with a hardcoded list of potential roots.
2. This is a global utility method, so `getProperCasing` could mean anything out of context. I'd change to `getTagCase` or `getProperTagCase`.